packages/synapse-react-client/src/components/AccessRequirementAclEditor/AccessRequirementAclEditor.stories.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button, Paper } from '@mui/material'
+import { Meta, StoryObj } from '@storybook/react'
+import React, { useRef, useState } from 'react'
+import {
+  MOCK_ACCESS_REQUIREMENT_WITHOUT_ACL_ID,
+  MOCK_MANAGED_ACCESS_REQUIREMENT_ACL,
+} from '../../mocks/mockAccessRequirementAcls'
+import { getAllAccessRequirementAclHandlers } from '../../mocks/msw/handlers/accessRequirementAclHandlers'
+import { getUserProfileHandlers } from '../../mocks/msw/handlers/userProfileHandlers'
+import { MOCK_REPO_ORIGIN } from '../../utils/functions/getEndpoint'
+import {
+  AccessRequirementAclEditor,
+  AccessRequirementAclEditorHandle,
+} from './AccessRequirementAclEditor'
+
+const meta: Meta<typeof AccessRequirementAclEditor> = {
+  title: 'Governance/AccessRequirementAclEditor',
+  component: AccessRequirementAclEditor,
+  render: function RenderFn(args) {
+    const [isSaving, setIsSaving] = useState<boolean>(false)
+    const ref = useRef<AccessRequirementAclEditorHandle>(null)
+
+    return (
+      <>
+        <Button
```

Comment:
Include a button in the story that can represent clicking "Save" in the SWC modal